charts/lagoon-logging/templates/logs-dispatcher.fluent-conf.configmap.yaml

```diff
@@ -318,7 +320,10 @@ data:
           # buffer params (per worker)
           total_limit_size 8GB
           # flush params
-          flush_thread_count 2
+          flush_thread_count 8 # 8 threads that flush, makes sure we have enough flushers for all the different buffers
+          flush_interval 2s # flush every 2 seconds
```

Comment:
Without the keepalive 2s might be too short for full TCP handshake and teardown. I wonder if we could bump this to 10s or more?